Packs/SplunkPy/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Splunk",
     "description": "Run queries on Splunk servers.",
     "support": "xsoar",
-    "currentVersion": "2.4.16",
+    "currentVersion": "2.4.17",
```

I'd suggest bumping it a little further...
```suggestion
    "currentVersion": "2.5.0",
```